website/src/website/data/playground-samples/extending-language-services/custom-languages/sample.js

```diff
@@ -30,12 +30,20 @@ monaco.editor.defineTheme("myCoolTheme", {
 
 // Register a completion item provider for the new language
 monaco.languages.registerCompletionItemProvider("mySpecialLanguage", {
-	provideCompletionItems: () => {
+	provideCompletionItems: (model, position) => {
+		var word = model.getWordUntilPosition(position);
```

Comment:
`range` is necessary.
But this sample is working before fix, so monaco.d.ts is strict.